### ISSUE TITLE
Scale image using alt text syntax

### ISSUE THREAD

**Issue by regalialong** [lun-4/obsidian2web#43]
Obsidian has its own syntax for scaling images by defining the resolution in the alt text of an image (ignoring how yucky that is), the syntax is described [here](https://help.obsidian.md/Linking+notes+and+files/Embed+files#Embed+an+image+in+a+note) in the docs but for brevity:
- ~~`![[image.extension|666]]` scale "according to its original aspect ratio"~~
  - ~~Obsidian sets `width` on image element to `666` for this~~
  - ~~This is technically not preferred for web content because this will cause a layout shift as width and height need to be defined for the browser to preallocate the space before the image loads in, but ¯\\\_(ツ)\_/¯~~
- ~~`![[image.extension|666x666]]` to set exact resolution~~
- ~~`![[image.extension|alt-text|666]]` allows adding alt text while also setting the width of the image~~
  - ~~o2w doesn't fail but (correctly) only adds the alt text currently when doing this~~
- Also valid for external locations per `![666](https://externalfilelocation.example/image.extension)` 

We don't always control the size of images, especially externally linked ones, so having a way to scale them would be useful.
